members/README.md

```diff
@@ -114,3 +114,26 @@ Creates new request for member introduction
 - **Error Response:**
     - **Code:** 500
         - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+## Patch /members/archiveMembers/:username
```

Comment:
Also please check as we are deprecating this route /members in favor of /users. Should we use this /members for new route or not please check.

Reply:
> The term "users" specify all user roles, including new users, members, and super users. Therefore, using `/users` instead of `/members` as the endpoint is more descriptive. This is because any authorized user on the server can archive a user, regardless of their role. Implementing this change would require extensive modifications to the codebase and should be considered as a separate task or ticket for the future.

`This is because any authorized user on the server can archive a user, regardless of their role` : Could you please kindly double-check this statement? I don't believe it is accurate.

Reply:
This means `any authorized user` refers to a superuser, only they have access to do such actions, and `regardless of their role` is meant to be regardless of any role whether a user is a member/ user. 

overall this statement states,  that any user with the appropriate permissions or access rights can perform the action of archiving another user's account, regardless of the roles or privileges the user being archived may have.